### PR TITLE
Upgraded compiling documentation with description of ELMAH 2x

### PR DESCRIPTION
**Pull request by ThomasArdal**
It may need some small tweaks, but would love to hear what you think so far.

### CODE REVIEW

**Comment by ThomasArdal** [comment]
@atifaziz Anything missing from my side to get this merged?

**Comment by atifaziz** [comment]
@ThomasArdal I added some comments to the commit but they seem to be gone now, perhaps because the history was rewritten? I'll try and add them again.

**Comment by ThomasArdal** [comment]
@atifaziz I think that you added them to the original commit back at the fork. Maybe you should add them at the PR? I already fixed a couple of things. I thank that the last thing were that you mentioned 1.2 SP3 and that I asked if I should add info about that.

File: Compiling.md [hunk 1]
```diff
@@ -5,23 +5,42 @@ permalink: a/compiling/
...
+
+### Compiling Interactively Using Microsoft Visual Studio
+
+Use the Microsoft Visual Studio solution named Elmah.sln found in the root directory. There are project files supplied for Microsoft Visual Studio 2013 under respective sub-directories.
+
+### Batch Compiling Using MSBuild
+
+The `build.cmd` script launches MSBuild to compile the solution from the command-line. The batch script detects the versions installed on your system and builds only using those versions that are found to be installed. The detection is very simple. It expects to find the various MSBuild versions to be installed in their known default locations. If you supplied a custom path during installation, you may have to adjust the script accordingly.
+
+## ELMAH 1x
```
Comment (atifaziz) [inline Compiling.md]: This should read “ELMAH 1.2 SP2 and Earlier”

**Comment by atifaziz** [comment]
I think we can squash 022cf1447aa2b88a3dccb97829504ec68053d31b into 95bb2591498ee3dac540deed7b958236c13ada8c. Also I think this PR should've been against a distinct branch instead of `master` (makes the workflow clear and easier to review, manage & clean). When I'll merge, your `master` will need re-writing.

**Comment by ThomasArdal** [comment]
I'm not sure I understand. Should I squash the two commits on my branch or do you want to do it back on elmah/www? Also, what about the PR and the branch. Do you want me to create a new PR against another branch?

**Comment by atifaziz** [comment]
@ThomasArdal The usual workflow is: create a branch in your fork, submit a PR _against_ that branch. It's acceptable to rewrite history of a PR branch because it's considered work in review until merged. It's not a huge deal in this case, thought it should be trivial for you but then realized you're on _master_.

**Comment by ThomasArdal** [comment]
@atifaziz I can do the following: create a new fork, create a branch, add the changes to that as a single commit, create the PR. Deal?

**Comment by atifaziz** [comment]
@ThomasArdal Or you could just to the following to fix up your repo:

```
git checkout master
git branch compiling
git reset --hard 6ceca087abb3d7b02428e601e9e98d3ba3e821ad
git push -f origin master
git push -u origin compiling
git checkout compiling
```

You don't need to squash everything into one commit. Just the one I asked.

**Comment by ThomasArdal** [comment]
@atifaziz Ah OK. I'll look into that. Thx!

**Pull request closed without merging**